.changeset/forty-turtles-protect.md

```diff
@@ -0,0 +1,5 @@
+---
+'xstate': patch
+---
+
+The `sendTo(actorRef, event)` action creator introduced in `4.27.0` was not accessible from the package exports.
```

Comment:
```suggestion
The `sendTo(actorRef, event)` action creator introduced in `4.27.0`, which was not accessible from the package exports, can now be imported just like other actions:

```js
import { sendTo } from 'xstate';
```
```

Reply:
@davidkpiano this should be more like:
```suggestion
The `sendTo(actorRef, event)` action creator introduced in `4.27.0`, which was not accessible from the package exports, can now be used just like other actions:

```js
import { actions } from 'xstate';
actions.sendTo
```
```

Reply:
This is what you have requested as a preferred option [here](https://github.com/statelyai/xstate/issues/2923#issuecomment-1007035960)

Note that applying this suggestion might be slightly unpredictable as it contains a triple backtick at the end and I'm unsure how to "escape" this properly here. It would be better to apply this manually.